packages/manager/apps/web/client/app/domain/zone/delete-all/domain-zone-delete-all.html

```diff
@@ -5,21 +5,11 @@
         data-wizard-on-finish="deleteAllZone"
         data-wizard-title="'domain_tab_ZONE_default_delete_all' | translate"
     >
-        <div
-            data-wizard-step
-            data-wizard-step-valid="!ctrl.loading && ctrl.canDeleteAllZone"
-        >
+        <div data-wizard-step data-wizard-step-valid="!ctrl.loading">
             <oui-spinner data-ng-if="ctrl.loading"></oui-spinner>
 
             <div data-ng-if="!ctrl.loading">
-                <div
-                    class="alert alert-warning"
-                    role="alert"
-                    data-translate="domain_configuration_zonedns_delete_all_blocked"
```

Comment:
Maybe we can consider removing the translation as well from here:
- packages/manager/apps/web/client/app/domain/translations/Messages_fr_FR.json

And let the script removing other occurrences across other languages later on.